_search-plugins/knn/knn-index.md

```diff
@@ -122,24 +174,28 @@ An index created in OpenSearch version 2.11 or earlier will still use the old `e
 }
 ```
 
-### Supported faiss encoders
+### Supported Faiss encoders
 
-You can use encoders to reduce the memory footprint of a k-NN index at the expense of search accuracy. faiss has
-several encoder types, but the plugin currently only supports *flat* and *pq* encoding.
+You can use encoders to reduce the memory footprint of a k-NN index at the expense of search accuracy. Faiss has
+several encoder types, but the plugin currently only supports `flat`, `pq`, and `sq` encoding.
 
 The following example method definition specifies the `hnsw` method and a `pq` encoder:
 
 ```json
-"method": {
-  "name":"hnsw",
-  "engine":"faiss",
-  "space_type": "l2",
-  "parameters":{
-    "encoder":{
-      "name":"pq",
-      "parameters":{
-        "code_size": 8,
-        "m": 8
+{
```

Comment:
Please revert this change because here we are just trying to show an example of method definition. We don't need to add vector type and dimension

Reply:
Reverted all requests to show only the method.